### PR TITLE
Additional data for IcarMovementArrivalEventResource

### LINKED ISSUE #432: IcarMovementArrivalEventResource: additional information about former events
In our HM-System and at least in one of our partners, there is the request to enter additional information about an arriving animal which has taken place bevor the ArrivalEventDateTime.
[PR](https://github.com/adewg/ICAR/pull/431)

### PR DESCRIPTION
**Pull request by AndreasSchultzGEA**
In our HM-System and at least in one of our partners, there is the request to enter additional information about an arriving animal which has taken place bevor the ArrivalEventDateTime.

### CODE REVIEW

**Review by erwinspeybroeck — COMMENTED** [review]
Andreas, I'm a bit reluctant to do so. There are other API's to get that information, and what about last lactation, last lactation index, last treatment, ....
If the fields are not mandatory and nobody expects them from us, then it is of course possible ...

**Comment by cookeac** [comment]
I acknowledge some of this is covered by a lactation, but that this may not be immediately available to the destination farm during a movement. I'm not convinced that the solution is to add these fields to the movement event. We should look at adding them as optional status extensions to `icarAnimalCoreResource`, which is supplied as the `animalDetail` field of the movement.
I'll put this as a comment in #432 as well.

File: resources/icarAnimalCoreResource.json [hunk 1]
```diff
@@ -81,6 +81,22 @@
           "$ref": "../enums/icarAnimalLactationStatusType.json",
           "description": "Lactation status of the animal."
         },
+        "parity": {
```
Comment (cookeac) [inline resources/icarAnimalCoreResource.json]:
```suggestion
        "currentLactationParity": {
```

Comment (AndreasSchultzGEA) [reply resources/icarAnimalCoreResource.json]: @cookeac, I changed as agreed on.

**Review by cookeac — APPROVED** [review]

**Review by erwinspeybroeck — APPROVED** [review]

**Comment by cookeac** [comment]
Resolves #432

**Pull request merged**